docs/PORTING.md

```diff
@@ -1,36 +1,43 @@
-# Porting How-To
+# Porting how-to
 
-This document describes the requirements and necessary steps required to port
-`mcuboot` to a new target `OS`.
+This document describes the requirements and necessary steps needed to
+port `MCUboot` to a new target `OS`.
```

Comment:
the `MCUboot`

Reply:
On this, I disagree, as it really doesn't sound well. Just substitute MCUboot with any proper name of software and it still wouldn't sound well.